calico-cloud/image-assurance/overview.mdx

```diff
@@ -0,0 +1,43 @@
+---
+description: About Calico Cloud Image Assurance.
+---
+
+## Big picture
+
+Scan images and Kubernetes workloads for vulnerabilities using {{prodname}} Image Assurance.
+
+## Value
+
+{{prodname}} Image Assurances helps you identify vulnerabilities in workloads that you deploy to Kubernetes clusters. A vulnerability can be a weakness in an application (for example, a design flaw or an implementation bug) that allows attackers to cause additional harm. With Image Assurance you can:
+
+- Scan an image for vulnerabilities
+- Assess the impact of newly-found vulnerabilities and prioritize remediation efforts
+- Create exceptions to ignore specific vulnerabilities
+- Create alerts on high-severity vulnerabilities so you can delegate remediation efforts to the appropriate team
+- Block non-compliant workloads using policy as part of your cloud-native security posture
+
+## About Image Assurance
+
+Image Assurance is based on the Common Vulnerabilities and Exposures (CVE) system, which provides a catalog of publicly-known security vulnerabilities and exposures. Known vulnerabilities are identified by a unique CVE ID based on the year it was reported (for example, CVE-2021-44228).
+
+Image Assurance provides a command-line interface (CLI) and cluster scanner to scan images, with the option to send results to the Manager UI. Scanned image content includes:
+
+- Libraries and content (for example, python, ruby gems, jars and go)
+- Packages (OS and non-OS)
+- Image layer
+
+## Before you begin
+
+**Update your cluster**
+
+Update your cluster to get the latest version of Image Assurance:
+
+1. Navigate to the **Managed Clusters** page.
+2. Select the cluster from the list, and click **Reinstall**.
+3. Copy the updated installation script command and run it against your cluster.
```

Comment:
In the How to section, delete these entries that no longer apply:

[Understand scan results in Manager UI](https://deploy-preview-676--tigera.netlify.app/calico-cloud/image-assurance/scanners/cli-based-scanner#understand-scan-results-in-manager-ui)
[Create vulnerability exceptions](https://deploy-preview-676--tigera.netlify.app/calico-cloud/image-assurance/scanners/cli-based-scanner#create-vulnerability-exceptions)
[Export scan results](https://deploy-preview-676--tigera.netlify.app/calico-cloud/image-assurance/scanners/cli-based-scanner#export-scan-results)